### PR TITLE
Ingela/ssl/hello pause/erl 169/otp 14372

### PR DESCRIPTION
**Pull request by IngelaAndin**
Allow user to inspect hello extensions and cancel or continue with possible new TLS options.
This aims to implement feature request discussed in ERL-169

### CODE REVIEW

**Comment by IngelaAndin** [comment]
Updated faulty dialyzer spec

File: lib/ssl/doc/src/ssl.xml [hunk 1]
```diff
@@ -213,6 +225,14 @@
       Defaults to <c>tls</c> Introduced in OTP 20, DTLS support is considered
       experimental in this release. DTLS over other transports than UDP are not yet supported.</p></item>
```
Comment (peterdmv) [inline lib/ssl/doc/src/ssl.xml]: ... **is** not yet supported.

Comment (IngelaAndin) [reply lib/ssl/doc/src/ssl.xml]: Transports is plural so I think it should  be are?!

Comment (ferd) [reply lib/ssl/doc/src/ssl.xml]: It depends on whether you consider that it is "DTLS over other transport" (singular) or "other transports" (plural). By the structure of the sentence, I'd agree with singular as it's a general structure that is not there, but if it's the other way around, I'd recommend rewording like:

> Other transports than UDP are not yet supported for DTLS.

Comment (IngelaAndin) [reply lib/ssl/doc/src/ssl.xml]: I will go for the rewording. The other transport most desired at the moment I think is SCTP but I think it is a good practice to not make limiting assumptions.

**Pull request merged**